libarchive/archive_write_disk_windows.c

```diff
@@ -1322,6 +1322,13 @@ restore_entry(struct archive_write_disk *a)
 		}
 	}
 
+    if ((en == ENOENT) && (archive_entry_hardlink(a->entry) != NULL)) {
+        archive_set_error(&a->archive, en,
+            "Hard-link target '%s' does not exist.",
+            archive_entry_hardlink(a->entry));
+        return (ARCHIVE_FAILED);
+    }
+
```

Comment:
Please adjust the indentation here to match the rest of the file.  (libarchive uses hard tabs for indents)